### ISSUE TITLE
influxdb_2.0.0-beta back and restore to 2.1.1 fails 'Error: no backup manifests'

### ISSUE THREAD

**Issue by rarrr** [influxdata/influxdb#23102]
__Steps to reproduce:__
List the minimal actions needed to reproduce the behavior.

1. influx backup -p ./ on source VM
2. on new VM run influx restore ./

__Expected behavior:__
DB to be restored

__Actual behavior:__
New VM shows:
Error: no backup manifests found at: xxxxxx

__Environment info:__

* System info: Linux 5.4.0-97-generic x86_64
* InfluxDB version: 
* New: InfluxDB 2.1.1 (git: 657e1839de) build_date: 2021-11-09T03:03:48Z
* old: InfluxDB 2.0.0-beta.16 (git: 50964d732c) build_date: 2020-08-07T20:18:07Z

Basically I run the backup on the old VM with:
`influx backup -p /xxxxx/backup/ -t xxxxxx`
This spits out 26 tsm files and a .bolt.
and on the new VM I run:
`influx restore /xxxxx/backup`
which gives:
`Error: no backup manifests found at "/mnt/media/backup"`

If I do:
`influx backup -portable -p /xxxxx/backup/ -t xxxxxx`
I get the same result.

I'm obviously missing the meta data but I can't find anything in the documentation about backing up meta data.

**Comment by loren-klemesrud-veritone** [comment]
I was testing out an Influx installation and I tried something similar.

Steps to reproduce:
List the minimal actions needed to reproduce the behavior.

    influx backup <path> --org  <org>
    influx restore <path>
    
Expected behavior:
DB to be restored

Actual behavior:

Error: no backup manifests found at: xxxxxx

Environment info:

    System info: Ubuntu 20.0.4
    InfluxDB version:
    InfluxDB 2.0.9
    Influx CLI 2.3.0 (git: 88ba346) build_date: 2022-04-06T19:30:53Z

**Comment by vspinu** [comment]
Seeing the same problem on Ubuntu 20.0.4 with:

```
nflux CLI 2.3.0 (git: 88ba346) build_date: 2022-04-06T19:30:53Z
InfluxDB v2.2.0 (git: a2f8538837) build_date: 2022-04-06T17:36:40Z
```

**Comment by andreyg42** [comment]
I had seen the same error on attempt to restore backup from Influx 2.0.4.

I also had an error during `influx backup -b ${bucket_name}`. It created archives, but then complained that bucket "{bucket_name}" was not found (it was there).

Specifying bucket by it's id with `--bucket-id ${bucket_id}`, instead of ` -b ${bucket_name}`, fixed both: backup was completed with no errors and restore worked, too.

**Comment by xcdr** [comment]
I have the same error, InfluxDB 2.6, want to restore with --org and --bucket...

**Comment by jeffreyssmith2nd** [comment]
@xcdr Do you have any logs/info on what happened when the backup failed? Is it reproducible or was it a one-off instance?

**Comment by xcdr** [comment]
> @xcdr Do you have any logs/info on what happened when the backup failed? Is it reproducible or was it a one-off instance?

Unfortunately not, but I suppose temporary out of disk space, not sure.

**Comment by jeffreyssmith2nd** [comment]
> I had seen the same error on attempt to restore backup from Influx 2.0.4.
@andreyg42 Did this get resolved for you?

> I also had an error during influx backup -b ${bucket_name}. It created archives, but then complained that bucket "{bucket_name}" was not found (it was there).

Was this a consistent error or something that happened once? My quick attempt to replicate it didn't exhibit the same issues.

**Comment by rarrr** [comment]
They say history repeats itself.  Once again I find myself trying to upgrade from influxd 2.0.0-beta.16 to  v2.7.1.  I did a backup from the old server, restore and the new one and got an error.  Googled the error and it's my issue from Feb 2022 as the top result.  I now remember why I gave up trying to upgrade it last time.

Is there another way I can get all my data across to the new version?  Since this doesn't seem to have a solution?

**Comment by rarrr** [comment]
if you're on 2.0.0-beta and getting that error, no I have not found a solution.  (Though someone above seems to have had success, I might re-visit)

Aside from that as far as I can tell the 2.0.0-beta is a dead end, it's somewhere between the v1 and v2 and doesn't have enough in common with either of them to work with either of them.

I've resided myself to the fact that I will either be stuck on 2.0.0-beta or will have to export and re import the data to the latest version somewhat manually.

**Comment by matteocampiotti** [comment]
I am only trying to back up with data that even if I lose it is not a problem so if I need it in the future with other data I will be prepared. 

I have influxdb in a docker container and made two attempts:
1. Backing up and then restoring to the same instance. It worked
2. Do the backup, delete the influxdb instance with its volumes, recreate the instance and do the restore. It gives me that manifest error.

So the version of influxdb from backup to restore is the same
